sdk/eventhub/Azure.Messaging.EventHubs.Shared/tests/Processor/PartitionLoadBalancerTests.cs

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Azure.Messaging.EventHubs.Tests;
+using System.Collections.Generic;
+
+namespace Azure.Messaging.EventHubs.Processor.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="PartitionLoadBalancer" />
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
+    public class PartitionLoadBalancerTests
+    {
+        private const string FullyQualifiedNamespace = "fqns";
```

Comment:
(preemptive comment)   Normally, we would ask for doc comments on each member, but in this case, I'd agree that they would be for ceremony only; these are pretty well self explanatory as to what they are, and I don't know that there would be much context in calling out how the tests use them.

The lone exception would be the cancellation source, which we would probably want to comment that it is used across tests and nobody should set cancellation.   That said, I'm hoping that we can consider removing that as a class member.

Reply:
Moved the cancellation source